screens/Menu/NearbyInterest.js

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import { View, Text, StyleSheet, FlatList, TouchableOpacity } from "react-native";
+import { Button } from "native-base";
+import { Feather } from "@expo/vector-icons";
+import PropTypes from "prop-types";
+
+/**
+ * US34 - As a user, I would like to see the nearest outdoor points of interest #14
+ * US35 - As a user, I would like to get the direction to the chosen nearest point of interest #15
+ * US36 - As a user, I would like to see a detailed description of the selected places #29
+ * 
+ * Description: This screen will presents the layout 
+ * for the display of the point of interests. The points 
+ * of interest information will be retrieved from the Google Places API.
+ * The screen is composed on a flatList to create a grid, and have 
+ * an pressable item for each points of interest
+ */
+
+export function formatData (data, numColumns) {
```

Comment:
1. Are you planning to use this function elsewhere? If so, keep the export, if not, remove it. 
2. And maybe arrow notation to be more consistent with how we did so far: i.e const formatData = () => {...}
3. On line 21, make sure _numColumns_ is greater than 0, because the code will crash if it is (even if we are never going to write 0).

**_4. Actually, what is the purpose of this method? It seems like the code works without that method._**

Very nice use of Flatlist ma mann 🤖

Reply:
The arrow notation you suggest was the way I had originally written the function. I'll bring back my original code. And yes, I'll put a condition for the _numColumns_ to prevent it from crashing

Reply:
The purpose of the method, was fixing a deprecated styling ("wrap") which set the size of the flat list container. Hence the reason, I had to added a _formatdata_ method was to keep size (width),  keep the consistency (2 elements per row) and handle the case of odd number of items